.gitignore

```diff
@@ -137,4 +138,10 @@ dist
 
 # Datastore files
 .data
+
 /prisma/dev.db
+
+
+#MacOS
+.DS_Store
+__MACOSX
```

Comment:
This is already done by #95 please revert the changes

Reply:
✅

Reply:
Can you try to run github action again? I think i can fix it in commit [64dc06b].